stock_location_orderpoint/models/stock_location_orderpoint.py

```diff
@@ -0,0 +1,423 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from copy import copy
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.osv import expression
+from odoo.tools import float_compare, split_every
+
+from odoo.addons.stock.models.stock_move import PROCUREMENT_PRIORITIES
+
+
+class StockLocationOrderpoint(models.Model):
+    _name = "stock.location.orderpoint"
+    _description = "Stock location orderpoint"
+    _order = "priority desc, sequence"
+
+    name = fields.Char(
+        "Name",
+        copy=False,
+        required=True,
+        readonly=True,
```

Comment:
Remove this one as:

- One may want to change the name manually
- With readonly, the call to next_by_code is done twice : one at clicking on 'Create', one at Save. As the value is not transmitted through the interface.

The other solution, maybe the best, is to add `force_save="1"` in views definition for the name field.